core/header.cpp

```diff
@@ -169,6 +169,15 @@ namespace MR
       if (App::project_version)
         H.keyval()["project_version"] = App::project_version;
 
+      std::string cmd = App::argv[0];
+      for (int n = 1; n < App::argc; ++n)
+        cmd += std::string(" \"") + App::argv[n] + "\"";
```

Comment:
Minor addition to this: Could probably use `Path::basename(App::argv[0])`. I'm not sure that keeping track of the location of the binary on the system for each command in the history is worth the bloat for the very rare instance where it might provide useful information.